pkg/controller/hyperconverged/factory.go

```diff
@@ -73,6 +78,19 @@ func newCDIForCR(cr *hcov1alpha1.HyperConverged, namespace string) *cdi.CDI {
 	}
 }
 
+func newMachineRemediationOperatorCR(cr *hcov1alpha1.HyperConverged, namespace string) *mrv1.MachineRemediationOperator {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
```

Comment:
What does creating this CR do?

Reply:
Creating all fencing controllers under the openshift-machine-api namespace.